### PR TITLE
fix(Accordion): deprecate (rename) expandBehaviour in favor of expandBehavior

### PR DESCRIPTION
**Pull request by langz**

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v12-info.mdx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+- [v12](#v12)
+  - [Migration](#migration)
+  - [Install](#install)
+  - [Component changes](#component-changes)
+    - [Accordion](#accordion)
+
+## Migration
+
+v11 of @dnb/eufemia contains _breaking changes_. As a migration process, you can simply search and replace:
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v12-info.mdx:15]:
```suggestion
v112of @dnb/eufemia contains _breaking changes_. As a migration process, you can simply search and replace:
```

File: packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v12-info.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+- [v12](#v12)
+  - [Migration](#migration)
+  - [Install](#install)
+  - [Component changes](#component-changes)
+    - [Accordion](#accordion)
+
+## Migration
+
+v112of @dnb/eufemia contains _breaking changes_. As a migration process, you can simply search and replace:
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/about-the-lib/releases/eufemia/v12-info.mdx:15]:
```suggestion
v12 of @dnb/eufemia contains _breaking changes_. As a migration process, you can simply search and replace:
```

File: packages/dnb-eufemia/src/components/accordion/AccordionDocs.ts [hunk 3]
```diff
@@ -134,9 +134,13 @@ export const AccordionProviderGroupProperties: PropertiesTableProps = {
     type: 'boolean',
     status: 'optional',
   },
-
   expandBehaviour: {
-    doc: 'Determines how many accordions can be expanded at once. Defaults to `single`',
+    doc: 'Determines how many accordions can be expanded at once. Defaults to `single`.',
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/accordion/AccordionDocs.ts:138]:
```suggestion
    doc: 'Use `expandBehavior` instead.,
```

**Review by tujoworker — APPROVED** [review]
🙏

**Pull request merged**